config/default.json

```diff
@@ -9,15 +9,24 @@
     "url": "mongodb://localhost/hearth"
   },
   "authentication": {
-    "authWindowSeconds": 10,
-    "type": "openhim-style",
+    "type": "disabled",
```

Comment:
I think this is a better default but it may affect people upgrading Hearth or not specifying a version...